test/Base_Tests/src/Data/Decimal_Spec.enso

```diff
@@ -97,15 +105,15 @@ add_specs suite_builder =
             mc4 = Math_Context.new 4
             mc5 = Math_Context.new 5
 
-            Problems.not_expect_warning (Decimal.new "123.25")
+            Problems.not_expect_warning Any (Decimal.new "123.25")
```

Comment:
`not_expect_warning` is especially useful if we do not expect a warning of type X but we acknowledge there may be other kinds of warnings.

If you expect no warnings at all, I'd recommend `Problems.assume_no_problems`

Reply:
Done.